src/applications/static-pages/download-1095b/components/App/index.unit.spec.js

```diff
@@ -48,57 +48,77 @@ describe('App component', () => {
   };
 
   describe('when not authenticated', () => {
-    it('renders the sign-in alert', () => {
+    it('renders the sign-in alert', async () => {
       store = mockStore(unauthenticatedState);
-      const { container } = render(
+      const { container, queryByText } = render(
         <Provider store={store}>
           <App />
         </Provider>,
       );
-      expect($('va-button', container).outerHTML).to.contain(
-        'Sign in or create an account',
-      );
+      await waitFor(() => {
+        expect(queryByText('Loading')).not.to.exist;
```

Comment:
We test for the absence of "Loading" multiple times. Does a test for "Loading" to exist make sense?

Reply:
Thanks! Looks like testing for its non-existence was redundant with the `waitFor` call on the next line, so I've removed the non-existence checks. If we need to test for the "Loading" message, I'll add it in the unit tests issue I'm working on.